### PR TITLE
FLPATH-330: Update the project status from the API

### PR DESCRIPTION
**Pull request by anludke**
Provide a quick fix as a continuation of [FLPATH-285](https://issues.redhat.com/browse/FLPATH-285) to dynamically update project given the current project and workflow execution mapping from the UI
Screenshot 2023-04-21 at 3 27 46 PM
Screenshot 2023-04-21 at 3 29 28 PM
Screenshot 2023-04-21 at 3 29 42 PM

### CODE REVIEW

**Review by RichardW98 — APPROVED** [review]

File: app-config.yaml [hunk 1]
```diff
@@ -61,7 +61,7 @@ proxy:
     headers:
       Content-Type: 'application/json'
       accept: 'application/json'
-      # Authorization: ${PARODOS_AUTH_KEY}
+      Authorization: ${PARODOS_AUTH_KEY}
```
Comment (mareklibra) [inline app-config.yaml:64]: Why is that change needed?
We should not use this anymore since we gather authentication at runtime.

File: plugins/parodos/src/components/projectOverview/ProjectOverview.tsx [hunk 2]
```diff
@@ -18,8 +18,8 @@ type ProjectFilters = ProjectStatus | 'all-projects';
 
 const projectFilterItems: { label: string; value: ProjectFilters }[] = [
   { label: 'All Projects', value: 'all-projects' },
-  { label: 'In Progress', value: 'in-progress' },
-  { label: 'On Boarded', value: 'on-boarded' },
+  { label: 'In Progress', value: 'IN_PROGRESS' },
```
Comment (mareklibra) [inline plugins/parodos/src/components/projectOverview/ProjectOverview.tsx:21]: We should unify the capital letters
`In Progress` vs `In progress`.
